src/index.js

```diff
@@ -45,7 +45,8 @@ export default function(h) {
         data = data || {}
         isDeclsFunction && (parsed = parse(decls(data)))
         var node = h(tag, data, children)
-        node.data.class = (node.data.class +
+        node.data.class = (
+          node.data.class || "" +
```

Comment:
@lukejacksonn @morishitter I was thinking, maybe we can instead check if `tag` is a function here, and call it directly with the new data, so that we don't have to access VNode's private data. Hyperapp's `data` (soon-to-be props), but it could be attrs or jollygood in another library AFIC.

Reply:
Sounds good.. would you want that in this PR? Might be worth merging this then opening a new one with an is function check.

Reply:
@lukejacksonn You are right. Maybe in a new PR after we have tests! Thanks.